index.js

```diff
@@ -102,8 +101,14 @@ const { parse, compileScript, compileTemplate } = require('@vue/compiler-sfc');
   }
 
   async function getCompositionApiSFC(source, template) {
-    const compiledScript = compileScript(parse(source).descriptor, {id: 'demo-'});
-    const compiledTemplate = compileTemplate({source: template ,id: 'demo-'});
+    const {vueCompilerSFC} = window;
```

Comment:
We need to obtain a globally available function/object that exposés the necessary (parse, compileScript, compileTemplate) functions, like we operate "window.translateSFC" (i.e., it may have a diff global name)l